pkg/kube/client/clientset/versioned/fake/clientset_generated.go

```diff
@@ -34,7 +34,7 @@ func NewSimpleClientset(objects ...runtime.Object) *Clientset {
 		}
 	}
 
-	cs := &Clientset{tracker: o}
+	cs := &Clientset{}
```

Comment:
This is not intentional, I regenerated those files and that was the only change. Probably doesn't matter?